_docs-sources/pipelines/tutorial/index.md

```diff
@@ -1,3 +1,159 @@
+  --var InfraLiveRepoName="infrastructure-live" \
+  --var GithubOrg="<your GitHub organization name>" \
+  --non-interactive
+```
+
+Push your changes to the `infrastructure-pipelines` repository you created in [Create repos in your org](#create-repos-in-your-org).
+
+### Infrastructure live
+
+Next, generate the `infrastructure-live` repository code using Boilerplate. Use the following command, replacing the values wrapped in `<>` with real values for your organization.
+
+```bash
+boilerplate --template-url "git@github.com:gruntwork-io/terraform-aws-architecture-catalog.git//blueprints/components/single-account-pipeline?ref=devops-foundations" \
+  --output-folder ./infrastructure-live \
+  --var AwsAccountName="<friendly name for your AWS account (e.g., dev)>" \
+  --var AwsAccountId="'<account id for your AWS account>'" \
+  --var AwsAccountEmail="<e-mail address associate with root user for account>" \
+  --var InfraLiveRepoName="infrastructure-live" \
+  --var InfraPipelinesRepoName="infrastructure-pipelines" \
+  --var GithubOrg="<your GitHub organization name>" \
+  --var OrgNamePrefix="<your organization name>" \
+  --non-interactive
+```
+
+Before pushing your changes, you will need to run an `apply` locally to provision the AWS IAM role that pipelines will use to deploy resources in your account. This should be the only time you need to manually run `apply` to provision resources for this account, moving forward pipelines will handle all the lifecycle of all resources for you, based on the code you commit to your repository.
+
+First, run a `plan` in the newly created `github-oidc-role` directory to see the resources that will be provisioned. Replace `<account name>` with the value you used for `AwsAccountName` in the boilerplate command above.
+
+```bash
+cd <account name>/_global/github-oidc-role
+terragrunt plan
+```
+
+Once you reviewed the new resources that will be created, run `terragrunt apply` to create the resources.
+
+Finally, push your changes to the `infrastructure-live` repository you created in [Create repos in your org](#create-repos-in-your-org).
+
+## Running your first pipeline
+
+Next you’ll create a resource in your AWS account using pipelines and GitOps workflows. you’ll define an infrastructure unit that creates AWS S3 bucket in your account, push your changes and create pull request (PR) to run a `plan` action, then run an `apply` action to create the bucket by merging your PR.
+
+### Adding a new S3 bucket infrastructure-unit
+
+First, create the file structure that will contain the infrastructure unit that defines an S3 bucket in your environment. Replace `<account name>` with the value you used for `AwsAccountName` when generating your `infrastructure-live` code in the [infrastructure live](#infrastructure-live) section.
+
+```bash
+mkdir -p <account name>/us-east-1/<account name>/data-storage/s3
+touch <account name>/us-east-1/<account name>/data-storage/s3/terragrunt.hcl
```

Comment:
Worth mentioning they can use whatever region?

Reply:
ah good catch, thank you!

Reply:
Fixed in [b725870](https://github.com/gruntwork-io/docs/pull/1114/commits/b72587013325cb2f89d57e780d3cbae8f1bfd1b1)